packages/components/src/Tooltip/Tooltip.module.css

```diff
@@ -0,0 +1,88 @@
+/* stylelint-disable selector-pseudo-class-no-unknown */
```

Comment:
Better way via stylelintrc? Or would this be the only file with `:global` use?

Reply:
I think this is fine. 🤷 

> Or would this be the only file with :global use?

My guess is that we'll eventually end up doing it again as we start building on top of other components that are not from Headless UI and require us to dig in and target a specific element that's not exposed to us. But I don't have any specific examples; it's just a feeling.

Reply:
Sounds good, I think we can add to stylelint when we have to get across that bridge